### ISSUE TITLE
Can't set grid_colour

### ISSUE THREAD

**Issue by jj-wohlgemuth** [franshej/CustomMatPlot#12]
Hi,

thanks for this tool. I'm very happy with it. I have a small issue. I can't seem to set the grid_colour. In the example below the grid will be white. I expected it to be red. Maybe someone could look into it?

```
 m_lnf.setColour(cmp::Plot::x_grid_label_colour, juce::Colours::white);
 m_lnf.setColour(cmp::Plot::y_grid_label_colour, juce::Colours::white);
 m_lnf.setColour(cmp::Plot::grid_colour, juce::Colours::red);
```

`CPMAddPackage("gh:franshej/CustomMatPlot#1.1.2")`
`cmp::SemiLogX m_plot;`

Cheers!

**Comment by franshej** [comment]
Thanks! fixed in c3f15f9cad505ef390173f18553a56eadfcc1ea0

**Comment by jj-wohlgemuth** [comment]
Super! Works here with ...

`CPMAddPackage("gh:franshej/CustomMatPlot#1.1.3")`
